glean-core/ios/Glean/Metrics/Ping.swift

```diff
@@ -15,10 +15,10 @@ public class Ping {
     let includeClientId: Bool
 
     /// The public constructor used by automatically generated metrics.
-    public init(name: String, includeClientId: Bool) {
+    public init(name: String, includeClientId: Bool, sendIfEmpty: Bool = false) {
```

Comment:
As for the Kotlin code, shouldn't this be provided by the parsers? Why should we provide a default for this and not for the client id one?

Reply:
As above